examples/calib_pulses.py

```diff
@@ -0,0 +1,124 @@
+from Instruments import GDS_scope
+import os
+import SpinCore_pp as spc
+from datetime import datetime
+import pyspecdata as psd
+import numpy as np
+
+my_exp_type = "test_equipment"
+assert os.path.exists(psd.getDATADIR(exp_type=my_exp_type))
+# {{{ import acquisition parameters
+config_dict = spc.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = spc.get_integer_sampling_intervals(
+    config_dict["SW_kHz"], config_dict["acq_time_ms"]
+)
+# }}}
+sqrt_P = config_dict["amplitude"] * np.sqrt(75)  # we have a 75 W amplifier
+prog_beta = np.linspace(10, 70, 50, endpoint=False)
+prog_p90s = prog_beta / sqrt_P
```

Comment:
makes list of betas and the p90s based on the assumption that the sqrt(P) = amplitude * sqrt(75)